Source/Python/WarpXWrappers.cpp

```diff
@@ -164,13 +164,13 @@ namespace
     {
         WarpX& warpx = WarpX::GetInstance();
         warpx.InitData();
-        if (warpx_py_afterinit) {
+        if ( warpx_callback_py_map.count("afterinit") ) {
```

Comment:
Avoiding implicit casts to bool to increase clarity for less experienced readers:
```suggestion
        if ( warpx_callback_py_map.count("afterinit") == 1u ) {
```
In C++20, we will finally have `bool std::map::contains(key)`...